src/components/Footer/styles.scss

```diff
@@ -0,0 +1,11 @@
+@import '../../styles/main.scss';
+
+.footer-container {
+  height: rem(32px);
+  background-color: $secondary;
+
+  @include flex-center();
+  @include respond-above(md) {
+    height: rem(72px);
```

Comment:
In this case, since you've a suggestion on this [LOC](https://github.com/Cobda/cobda-web/pull/3/files#r561194816), you should also move this comment to that one too. Prevent opening multiple comments with same context 😎